src/api/app/assets/stylesheets/webui2/repositories.scss

```diff
@@ -11,3 +11,8 @@ details[open].repository-path {
   .more{ display: none; }
 }
 
+// Ensure repository footer has same height when being empty
```

Comment:
Wait... it should be empty... where is the `Request repository deletion` for the DoD? :scream: